packages/loader/container-loader/package.json

```diff
@@ -69,7 +69,7 @@
 		"@fluidframework/core-interfaces": "workspace:~",
 		"@fluidframework/driver-definitions": "workspace:~",
 		"@fluidframework/driver-utils": "workspace:~",
-		"@fluidframework/protocol-base": "^1.0.0",
+		"@fluidframework/protocol-base": "^2.0.0-181438",
```

Comment:
i don't think you can make this change. if you want to bump the server version it needs to be done in next, and for all server packages, not just a single one. I also believe when we take a dependency on a dev package like this we don't do semver, and if you use semver, like `^` a future breaking change can break things at runtime.